### ISSUE TITLE
Website no longer syncing. 

### ISSUE THREAD

**Issue by vpakg** [cpfair/tapiriik#155]
I believe something is wrong with the tapiriik website, maybe the processes jammed because it has been syncing for the last 2 days (literally) between endomondo and Strava. There is only 1 workout left to be synced. 

![sync](https://cloud.githubusercontent.com/assets/4654583/8801943/0821968e-2f8c-11e5-8b44-609957b0aec5.png)

**Comment by cpfair** [comment]
I can't diagnose individual accounts on GitHub, unfortunately. Please use the official contact page.

**Comment by jgrandscrutton** [comment]
I found that the site had me "queuing" all morning, and wasn't even trying to sync anything...

Sent using OWA for iPad
________________________________

**Comment by vpakg** [comment]
@cpfair Thanks for the response, I don't know if this is an account specific issue, tho. I've just tried with another service and I'm experiencing the same issue.

**Comment by cpfair** [comment]
It is account specific - adding or removing accounts doesn't touch job state.

**Comment by trepmal** [comment]
I'm also getting a neverending "queuing" - between Runkeeper and Garmin in my case. Tried disconnecting and reconnecting but that didn't resolve it. Please let me know if you'd like more specific info from me.

**Comment by vpakg** [comment]
The issue affect many users and I believe it should be reopened.

**Comment by cpfair** [comment]
I'm on my phone with sketchy Internet for the next while, so I can't check what's going on for sure. It looks like there might be a more widespread issue though.

**Comment by The-Maso** [comment]
+1 Stuck on queuing for two days between Runkeeper, Endomondo and Strava. Tried disconnecting and reconnecting to no avail. Still disc spinning, queuing.

Any ideas...?

**Comment by cpfair** [comment]
Database server is resynchronizing now, sync will resume some time after that.

**Comment by The-Maso** [comment]
Thanks, but still queuing - how long do you think is right?

Cheers...

**Comment by vpakg** [comment]
Still queuing for me as well.

**Comment by LimeBlast** [comment]
Sorry if this is the right place for this, but I'm also experiencing the Queuing issue.

**Comment by cmmata** [comment]
Is this bug solved? I'm having the same issue, yesterday morning I tried to sync and this morning it was still queuing. I have disconnected all accounts and connected again, but it seems the same. How many days do you think it can be usual to wait?

**Comment by vpakg** [comment]
It was solved back when I opened the issue. Apparently it is back.

**Comment by salomvary** [comment]
Queueing forever for me too. Strava id 3963318.

**Comment by EndeNeu** [comment]
Is this issue still actual? I'm experiencing the same problem, I've been in a queue for 2 hours and counting.

**Comment by rwa** [comment]
Syncing for hours again.

**Comment by a1westwood** [comment]
Been stuck at queuing for a couple of days now :-(

**Comment by DonStefano** [comment]
Same for me, no sync since Oct. 15th. Manual sync worked, when I decided to go payed.

**Comment by khlee888** [comment]
Me too. no sync since 21 Oct.

**Comment by a1westwood** [comment]
Something been fixed or reset :-)  I had to reauthorise Garmin Connect this morning, and that got me going again. Thanks Collin!!

**Comment by rwa** [comment]
A commit was pushed 24 hours ago but still no observable change for me.  Queuing infinitely...

**Comment by EndeNeu** [comment]
For me it takes a while (like 3 or 4 hours) but eventually it does sync (using Strava and Runkeeper).

**Comment by kanokc** [comment]
I've setup Endomondo - Runkeeper - Strava.   My last sync was on Oct 17 and I have activities on 23 and 25 which still waiting in Strava.

**Comment by cpfair** [comment]
Sorry - [maintenance at my main hosting provider](http://status.linode.com/incidents/ltchxw3jmx0s) meant the entire database cluster had to be rebooted late last week - not normally a problem since it's replicated, but filesystem issues prevented a secondary from coming up again. This uncovered the bug that @rwa saw the commit to fix: the scheduler was blocked waiting for database changes to be replicated to that (down) secondary. This bug left a bunch of users in limbo, "scheduled" in the database but not actually present in the message queue - I just reset everything, so things should be back on the rails. Sorry for the confusion.

**Comment by kanokc** [comment]
Everything is in-sync again..  

Thanks

**Comment by desrod** [comment]
Still not working for me. Seems to stop at October 12th, with spotty titles and data across them. I've disconnected and reconnected my services and re-sync'd many times, no luck, including after yesterday's email from Colin. 

![screenshot 2015-10-26 09 35 50](https://cloud.githubusercontent.com/assets/5942125/10730211/fb5dcb9a-7bc4-11e5-9e8a-11723346b2f7.png)

**Comment by TimMifsud** [comment]
Has this bug happened again?

**Comment by EndeNeu** [comment]
Since @cpfair posted I haven't had any issue.

**Comment by TimMifsud** [comment]
Thanks @EndeNeu. It's not working for me at the moment.

**Comment by freak4pc** [comment]
Queuing (Approx. 3 hours) ... Huh O.o

**Comment by TimMifsud** [comment]
Yeah I think there has been an issue. See the Tapiriik Twitter feed

**Comment by farajfarook** [comment]
I'm stucked in queueing as well. Any updates?

**Comment by theinmostdens** [comment]
Stuck in queuing, me too. From Garmin Connect to Runkeeper and Strava, but just stuck in queue. If it helps, I'm a paying user.

**Comment by rwa** [comment]
I've been stuck queuing for about 14 hours now, too.

**Comment by mindrunner** [comment]
+1

**Comment by thehandcoder** [comment]
I'm stuck as well.  I was considering using the service so I set it up for a test run.  Right now it's stuck queuing so I can't really evaluate.

**Comment by jhpmatos** [comment]
+1

**Comment by MissMaximas** [comment]
+1 Stuck in queuing

**Comment by datelligence** [comment]
There is no manual or auto synch button:  only connected account from smashrun,  no way to manually synch.

**Comment by rammuraly** [comment]
I have been stuck at queuing for the past 36 hours or so? any idea what's going on?

**Comment by tulkasEldarion** [comment]
+1 Stuck in queuing
